### DISCUSSION TITLE
MW4 Api

### DISCUSSION THREAD

**Discussion by WTRVGL** [mworion/MountWizzard4#255] [Q&A] [answered]
I was wondering if there is a way to interact with MW4 to automate the building of models? For example run a Pre-Event SGP script to build a model before running a sequence?

**Comment by mworion** [comment, marked as answer]
Hi,
right now there is hardly any remote feature there. If you need this feature, please have a look to Martin's Model Creator, which has a CLI method to build a model. MW4 is more intended to be a side Application always running during your Imaging time. I put it in the wish list.
Michel
